canvas/views/views.py

```diff
@@ -78,3 +78,26 @@ def events_options_view(request):
     return render(request, 'canvas/course_event_options.html', {
         'events': course.events.all(),
     })
+
+
+def create_event_view(request, pk):
+    course = get_object_or_404(CanvasCourse, pk=pk)
+    course.is_instructor(request.user)
```

Comment:
I don't understand why this line is here. I think it doesn't do anything. If so please remove this line.